app/components/access-level-pill.hbs

```diff
@@ -41,21 +41,22 @@
         />
       {{/if}}
     {{/if}}
-    <div class="auk-u-position-relative auk-u-inline-block">
-      <Auk::Button
-        @skin={{if @confidential "danger-borderless" "muted-borderless"}}
-        @layout="icon-only"
-        @icon={{if @confidential "lock-closed" "lock-open"}}
-        @disabled={{not this.session.isEditor}}
-        {{on "click" (perform this.changeConfidentiality)}}
-      />
-      {{#if @confidential}}
-        <EmberTooltip @side="top" @tooltipClass="tooltip-custom">
-          <p>
-            {{t "document-is-confidential"}}
-          </p>
-        </EmberTooltip>
-      {{/if}}
-    </div>
+    <Auk::Button
+      @skin={{if @confidential "danger-borderless" "muted-borderless"}}
+      @layout="icon-only"
+      @icon={{if @confidential "lock-closed" "lock-open"}}
+      @disabled={{not this.session.isEditor}}
+      {{on "click" (perform this.changeConfidentiality)}}
+    >
+      <:floater>
+        {{#if @confidential}}
```

Comment:
Kan de `if` hier niet rond de floater in z'n geheel?

Reply:
Zonet geprobeerd maar dat geeft de volgende error: `Syntax Error: Unexpected named block <:floater>`.

Vermoedelijk een limitatie binnen `ember-named-blocks-polyfill` of de manier waarop named blocks werken?